source/simulator/assembly.cc

```diff
@@ -457,14 +454,9 @@ namespace aspect
 #ifdef ASPECT_USE_PETSC
     Amg_data.symmetric_operator = false;
 #else
-#if DEAL_II_VERSION_GTE(9,2,0)
     Amg_data.constant_modes = constant_modes;
-#else
     // To avoid a Trilinos error, only define constant modes
     // if this mpi rank owns any DoFs:
```

Comment:
I should have been more precise: These two comment lines also need to go, they belong to the code you removed in the last commit.